ui/pages/confirm-send-token/confirm-send-token.component.js

```diff
@@ -12,8 +12,9 @@ export default class ConfirmSendToken extends Component {
 
   handleEdit(confirmTransactionData) {
     const { editTransaction, history } = this.props;
-    editTransaction(confirmTransactionData);
-    history.push(SEND_ROUTE);
+    editTransaction(confirmTransactionData).then(() => {
```

Comment:
In `ui/pages/confirm-send-ether/confirm-send-ether.container.js` we made a change to use `editExistingTransaction` instead of `editTransaction`. but not here

was that intentional?

Reply:
the container wraps editExistingTransaction and calls it 'editTransaction' i'm changing this to avoid confusion and making debugging better.